Add new doc about linking OTEL apps with K8s (#7475)
newrelic/docs-website: src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-apm-applications-kubernetes.mdx

  - /docs/integrations/kubernetes-integration/metadata-injection/metadata-injection
  - /docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection
  - /docs/kubernetes-pixie/kubernetes-integration/link-your-applications/link-your-applications-kubernetes
  - /docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-your-applications-kubernetes
---

> You can surface Kubernetes metadata and [link it to your APM agents](/docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection#agent-compatibility) as [distributed traces](/docs/understand-dependencies/distributed-tracing/get-started/introduction-distributed-tracing) to explore performance issues and troubleshoot transaction errors. For more information, see this [New Relic blog post](https://blog.newrelic.com/engineering/monitoring-application-performance-in-kubernetes/).

Reviewer: ```suggestion
You can surface Kubernetes metadata and [link it to your APM agents](/docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection#agent-compatibility) as [distributed traces](/docs/understand-dependencies/distributed-tracing/get-started/introduction-distributed-tracing) to explore performance issues and troubleshoot transaction errors. For more information, see [this blog post about monitoring app performance via Kubernetes](https://blog.newrelic.com/engineering/monitoring-application-performance-in-kubernetes/).
```
